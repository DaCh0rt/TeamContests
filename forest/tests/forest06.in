3 510
-9 -9
-9 -10
-10 -9
