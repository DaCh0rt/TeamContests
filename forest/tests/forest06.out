3207.84 818870.00
